cf_units/tests/integration/test__Unit_date2num.py

```diff
@@ -0,0 +1,41 @@
+# Copyright cf-units contributors
+#
+# This file is part of cf-units and is released under the BSD license.
+# See LICENSE in the root of the repository for full licensing details.
+"""Test method :meth:`cf_units.Unit.date2num`."""
+
+
+import cftime
+import pytest
+
+import cf_units
+
+CALENDAR_CONSTANTS = [
+    cf_units.CALENDAR_STANDARD,
+    cf_units.CALENDAR_360_DAY,
+    cf_units.CALENDAR_365_DAY,
+]
+
+CALENDAR_STRINGS = ["standard", "360_day", "365_day"]
```

Comment:
`CALENDAR_CONSTANTS` and `CALENDAR_STRINGS` are literally the same since the constants are just strings.  Wasn't really sure whether it's worth specifying them separately line this, but it seems to be the convention in the other tests that you pass the constant to `cf_units` and the string to `cftime`.

Reply:
I'm okay with this :+1: